statsmodels/discrete/conditional_models.py

```diff
@@ -13,18 +13,27 @@
 import itertools
 
 
-
 class conditionalModel(base.LikelihoodModel):
 
     def __init__(self, endog, exog, missing='none', **kwargs):
 
         if "groups" not in kwargs:
-            raise ValueError("groups is a required argument")
+            raise ValueError("'groups' is a required argument")
```

Comment:
this line is better than the status quo, but better yet would be to make `groups` explicit in the signature instead of part of `**kwargs`

Reply:
I did this to match the signature of `base.LikelihoodModel.__init__` and thereby keep the CI linting tools from complaining about it.